ignite/contrib/handlers/neptune_logger.py

```diff
@@ -78,7 +78,7 @@ class NeptuneLogger(BaseLogger):
                 project_name="shared/pytorch-ignite-integration",
                 experiment_name="cnn-mnist", # Optional,
                 params={"max_epochs": 10}, # Optional,
-                tags=["pytorch-ignite","minst"] # Optional
+                tags=["pytorch-ignite","mnist"] # Optional
```

Comment:
Could you please send a separate PR for this typo. It will be merged more quickly and wont require discussions and multiple iterations as this PR... Thanks !